version/version.go

```diff
@@ -1,5 +1,5 @@
 package version
 
 var (
-	Version = "1.8.3"
+	Version = "1.9.0"
```

Comment:
The installer PR is referring to 1.7.6, does it need to be updated? (https://github.com/integr8ly/installation/pull/926/files#diff-15e7e03f5a684bad56e00d52ba85f461R67)

Reply:
Installer has correct version. I will have to backport this to 1.7.x because in 1.8.3 there some alerts for Integreatly 1.6, which we don't want in Integreatly 1.5.
So once this is merged, I'll send cherry-pick PR for v1.7 branch.